locales/en-US/app.ftl

```diff
@@ -227,10 +227,15 @@ Home--menu-button-instructions =
     Enable the profiler menu button to start recording a performance
     profile in { -firefox-brand-name }, then analyze it and share it with profiler.firefox.com.
 
-Home--addon-button = Install add-on
-Home--addon-button-instructions =
-    Install the Gecko Profiler Add-on to start recording a performance
-    profile in { -firefox-brand-name }, then analyze it and share it with profiler.firefox.com.
+Home--enable-button-unavailable =
+    .title = This profiler instance was unable to connect to the WebChannel, so it cannot enable the profiler menu button.
+
+Home--web-channel-unavailable =
+    This profiler instance was unable to connect to the WebChannel. This usually means that it’s
+    running on a different host from the one that is specified in the preference
+    <code>devtools.performance.recording.ui-base-url</code>. If you would like to capture new
+    profiles with this instance, and allow programmatic control of the profiler menu button from
```

Comment:
Maybe we can avoid repeating "from this instance" here? 

> If you would like to capture new profiles with this instance, and give it programmatic control of the profiler menu button, you can go to…

ubernit: we talk about "settings" these days, not "preferences", so maybe it's worth using that word here.

Reply:
about:config's warning page says:

> Changing advanced configuration preferences can impact Nightly performance or security.
> [ ] Warn me when I attempt to access these preferences

And then in the about:config page:
![Capture d’écran de 2021-08-05 14-13-38](https://user-images.githubusercontent.com/454175/128347995-fd47f507-9c2d-4e46-9144-05aaf3c3a10b.png)

So I would say that in this case we really want "preference".

Reply:
Interesting, I guess we haven't got there yet. I agree we should use "preference" then.

Reply:
Could that be the difference between a "setting" (that's what the user will set by clicking checkboxes in the settings page (which is about:preferences 😁)) and a "preference" (that's the technical thing behind all this) ?

Reply:
> Maybe we can avoid repeating "from this instance" here?
> 
> > If you would like to capture new profiles with this instance, and give it programmatic control of the profiler menu button, you can go to…

Great suggestion, thanks!